web_client/models/AnnotationModel.js

```diff
@@ -39,6 +49,55 @@ export default Model.extend({
     },
 
     /**
+     * Fetch a single resource from the server. Triggers g:fetched on success,
+     * or g:error on error.
+     * To ignore the default error handler, pass
+     *     ignoreError: true
+     * in your opts object.
+     */
+    fetch: function (opts) {
```

Comment:
I could submit a PR to Girder to fix that, which would allow us to use the super here...